app/views/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,12 @@
+<% if @user.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      <%= t ".text_div_1" %> <%= pluralize(@user.errors.count, t(".error")) %>.
```

Comment:
https://viblo.asia/p/i18n-in-rails-part-2-GrLZDAO3lk0
Nghiên cứu Cách thể hiện số ít và số nhiều với i18n để áp dụng chỗ này em nhé